scripts/test-proxy-cert.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+mkdir -p /etc/rancher/ssl/
```

Comment:
Will this run locally? Potential permission issues doing this and it's going to pollute someones system

Reply:
This is called from the `scripts/test.sh` file, which as far as I can tell is only called from the `make ci/build` command, which would be inside the dapper/docker container.

Reply:
A better place would be the management-state dir we put stuff in already